admin/src/templates/header.html

```diff
@@ -4,9 +4,27 @@
       <h1 translate>admin.app.name</h1>
     </div>
     <div class="extras">
-      <a href="#" ng-click="logout()">
-        <span translate>Log Out</span>
-      </a>
+      <span class="dropdown options">
+        <a data-toggle="dropdown">
+          <div class="mm-icon mm-icon-inverse">
+            <span class="fa fa-fw fa-bars"></span>
+          </div>
+        </a>
+        <ul role="menu" class="dropdown-menu mm-dropdown-menu">
+          <li role="presentation" mm-auth="can_configure">
```

Comment:
Remove the `mm-auth` attribute - that's required to view the admin app, not the webapp.